geojson_pydantic/types.py

```diff
@@ -11,18 +11,14 @@
 Position = Union[Tuple[float, float], Tuple[float, float, float]]
 
 # Coordinate arrays
-
 if TYPE_CHECKING:
-    MultiPointCoords = List[Position]
     LineStringCoords = List[Position]
-    MultiLineStringCoords = List[List[Position]]
     LinearRing = List[Position]
-    PolygonCoords = List[List[Position]]
-    MultiPolygonCoords = List[List[List[Position]]]
 else:
-    MultiPointCoords = conlist(Position, min_items=1)
     LineStringCoords = conlist(Position, min_items=2)
```

Comment:
I tried to make these `Annotated[List[Position], Field(min_items=2)]` which works for `LineStringCoords` but does not properly constrain `MultiLineStringCoords`. I think that is something that may get fixed in Pydantic 2.